packages/ui-focusable/src/Focusable/README.md

```diff
@@ -57,83 +57,3 @@ example: true
 }}
 </Focusable>
 ```
-
-The `FocusableView` component can be used with `Focusable` to render a minimal focus outline when
```

Comment:
Running a quick search in the code, I could find a few other usages of `FocusableView` in the code: 
- packages/\_\_docs\_\_/components.js
- packages/ui-focusable/README.md
- rfcs/Button/variants.md

(also in v6 codemods, but that probably should stay)

Can you please have a look if any of this is needed?

Reply:
Good catch, I've removed the first 2; the third one is an RFC, this is meant as a historic document that should not be changed